types/react-jsonschema-form/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-jsonschema-form 1.3
+// Type definitions for react-jsonschema-form 2.0
```

Comment:
[Latest published version](https://www.npmjs.com/package/react-jsonschema-form) is 1.6 I think

Reply:
Yes, it's true. I thought that was the definition type version :/

I'll change it. 👍